src/blueapi/service/model.py

```diff
@@ -174,6 +174,25 @@ class EnvironmentResponse(BlueapiBaseModel):
     )
 
 
+class RepositoryStatus(BlueapiBaseModel):
+    remote_url: str = Field(description="URL of a repository on a remote server")
+    version: str = Field(description="Branch or tag of the repository")
+    is_dirty: bool = Field(description="Does the repository have uncommitted changes")
+
+
+class ScratchResponse(BlueapiBaseModel):
```

Comment:
Should: Include whether scratch area is enabled at all (i.e. if scratch config is not `None`), might be scope creep in which case feel free to make a ticket.